### ISSUE TITLE
Wiki: MQTT, Websocket, and Client Event Agent pages creation/update

### ISSUE THREAD

**Issue by DonWillems** [openremote/openremote#377]
- [ ] Review and update Websocket page: https://github.com/openremote/openremote/wiki/User-Guide%3A-Connecting-to-a-Websocket-Server
- [ ] Review and update MQTT page: https://github.com/openremote/openremote/wiki/How-to-work-with-MQTT
- [ ] Create Client Event Agent page explaining the multiple uses of this agent.
- [ ] Review the 'Websocket Client Agent' and 'Client Event Agent' available in the platform (what are the differences?)

**Comment by pierrekil** [comment]
@Miggets7 can you test the flow using the UI on the Manager? Seems fields missing and/or not showing values when e.g. creating a ClientEvent Agent via the UI

**Comment by senfgurke** [comment]
Hello,

I can confirm with a clean docker-compose.yml installation on a docker desktop system (windows), it is not possible to establish a mqtt connection to openremote.

clients:  MQTTX (windows), esp32 , MQTT Explorer

Even in the logs of the containers no log failoure occours.
There is something strange in the neighborhood :)

It would be great if the documentation can be updated or a troubleshoot section  is added.

Basically this is the nicest MQTT broker I've ever seen.

Why is the docker-compose.yml is not bring up a out of the box working solution. Before we host the system on the server, of course we like to play and test :)

It would be great if you can reproduce the problem and give some hints whats wrong.
Thank you very much.

**Comment by amanas** [comment]
Same problem here. Openremote seems really awesome, except that Matt connectivity is not working. In my case that means I can not use this platform at all.

**Comment by richturner** [comment]
Hi,

Thanks for taking the time to provide feedback; we have a few documentation updates pending for our Manager APIs (HTTP, WS and MQTT) I'm hoping to find time the next day or so and once complete I'll put something here.

Rich

**Comment by amanas** [comment]
Thanks for your effort, Rick. After playing for a while with openremote, if mqtt would work fine, it would be difficult not to choose this platform for any IoT project.

**Comment by aktur** [comment]
MQTT is working OK as it is running in prod just fine. The only thing missing is indeed documentation and it is a shame that we can discourage potential users by lack of it.

**Comment by amanas** [comment]
Thanks @aktur, sorry for wrong description; there is a lack of knowledge on my side.

In case it could be of help for anybody, this [forum](https://forum.openremote.io/t/help-sensor-raspberry-pi-openremote/743) could be related.

**Comment by amanas** [comment]
Hello, any progress with the documentation?
